go.mod

```diff
@@ -8,22 +8,21 @@ require (
 	github.com/hashicorp/hcl v1.0.1-0.20190611123218-cf7d376da96d // indirect
 	github.com/imdario/mergo v0.3.8 // indirect
 	github.com/newrelic/infra-integrations-sdk v1.0.1-0.20201016141521-6fdfe28ad6c0
```

Comment:
Just a small comment tho, have we tried to upgrade this and see what breaks?

Reply:
It would seem that [v3.0.0](https://github.com/newrelic/infra-integrations-sdk/blob/master/CHANGELOG.md#300-beta) onwards removed protocol v1, so let's hold this for now.